tests/test_awslambda/test_policy.py

```diff
@@ -0,0 +1,104 @@
+from __future__ import unicode_literals
+
+import unittest
+import json
+
+from moto.awslambda.policy import Policy
+
+
+class MockLambdaFunction:
+    def __init__(self, arn):
+        self.function_arn = arn
+        self.policy = None
+
+
+class TC:
```

Comment:
Can you rework the tests to be more inline with the rest of Moto?
Typically we just have separate methods per test case, i.e.:

def test_add_statement():
    # setup
    # add statement
    # assert ...

Reply:
We'll get it swapped to one execution of all possible policy options instead of a table driven format.